loadData.sh

```diff
@@ -7,6 +7,12 @@ log() {
   echo "$(date +'%Y-%m-%d %H:%M:%S') - $1"
 }
 
+warm(){
+  VIEW=$1
+
+  curl -fsS $COUCHDB_URL/_design/views/_view/$VIEW >/dev/null
```

Comment:
One option to reduce the cost of this query might be to run it immediately after the creation of the view when. This would require a higher degree of coupling with the test data however and I'm not sure I like the tradeoff considering that load data on a new DB should be pretty infrequent.